uat/ReadmeUAT.md

```diff
@@ -0,0 +1,23 @@
+User Acceptance Tests (UATs)
+UATs are defined under uat module. 
+UATs use aws-greengrass-testing-standalone as the test framework to run the tests.
+aws-greengrass-testing-standalone is pulled as maven dependency from GG maven repo. 
+You can add/update UATs at uat source. 
+The uat module generates a UAT artifact (nucleus-uat-artifact.jar) which is an executable jar meant to run the UATs.
+
+Running UATs locally
+UAT runs require the credentials for the AWS account you want to use. 
+Ensure credentials are available in the environment. 
+Command to run UATs locally from the project root is:
```

Comment:
You are not describing here the fact that in order for this to work it requires an uber jar containing log manager + OTF.